src/components/common/ListItem.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import styles from './ListItem.module.css'
+import { Button } from './Button'
+
+export const ListItem = ({ id, creditorId, amount, paidAt, users }) => {
+  const [isPaid, setIsPaid] = useState(paidAt)
+
+  const printPrize = (amount) => amount ? amount.toLocaleString('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 }).replace('.', ',') : 'N/A'
+
+  const checkPaid = () => isPaid === null ? '' : styles.lineThrough
```

Comment:
I wouldn't wrap this in a function as it's going to be executed anyhow. This just adds unnecessary overhead.
```js
const checkPaid = isPaid === null ? '' : styles.lineThrough
```